deployment/ccip/changeset/utils.go

```diff
@@ -0,0 +1,28 @@
+package changeset
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+)
+
+type WrappedChangeSet[C any] struct {
+	operation deployment.ChangeSetV2[C]
+}
+
+// RunChangeset is used to run a changeset in another changeset
+// It executes VerifyPreconditions internally to handle changeset errors.
+func RunChangeset[C any](
```

Comment:
This should be moved to deployment package. Also please add some comment when this might be needed

Reply:
i couldn't find existing way to call one changeset from another in changesetV2 model so had to use this